RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/Playback/PlaybackScreen.java

```diff
@@ -400,20 +419,76 @@ private void setButtonHandlers() {
         mInsertBtn.setOnClickListener(btnClick);
         mSwitchToMinimap.setOnClickListener(btnClick);
         mSwitchToPlayback.setOnClickListener(btnClick);
-        mVerseMarkerMode.setOnClickListener(btnClick);
+        mEnterVMarkerMode.setOnClickListener(btnClick);
+        mExitVMarkerMode.setOnClickListener(btnClick);
+        mDropVMarkerBtn.setOnClickListener(btnClick);
+        mCompleteVMarkerBtn.setOnClickListener(btnClick);
     }
 
     private void enableButton(int id, boolean isEnable) {
         findViewById(id).setEnabled(isEnable);
     }
 
     private void enableButtons() {
+        // NOTE: Why do we need to enable these buttons?
         enableButton(R.id.btnPlay, true);
         enableButton(R.id.btnSave, true);
     }
 
-    private View.OnClickListener btnClick = new View.OnClickListener() {
+    private boolean allVersesMarked() {
+        return mVerses <= 0;
+    }
+
+    private int getVerseCount() {
+        // NOTE: Replace with real code to get the number of verses in a chunk
+        int verses = 3;
+        // -1 because the first verse marker should be dropped at the beginning automatically
+        return verses - 1;
+    }
+
+    private void setVMarkerCount(int count) {
```

Comment:
Should we stick with a consistent naming convention here?
line 442 uses the full word Verse, here it is only V

I'd lean towards spelling out the word due to the need for plural in 438?